tf2_tools/CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog for package tf2_tools
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.0.0 (2019-11-04)
+-------------------
+ahcorde: Adding support for ROS 2.0
+
```

Comment:
Since this package hasn't been released yet it should have an entry in the changelog file yet (or only a section named `Forthcoming`).

Reply:
Its probably best to remove the new changelog section. It will be auto-generated by `catkin_generate_changelog` before the next release based on commit messages.